migration/migration_test.go

```diff
@@ -0,0 +1,159 @@
+package migration
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/query"
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/powergate/tests"
+)
+
+func TestEmptyDatastore(t *testing.T) {
```

Comment:
Here some tests of general `Migrator` library.
In this test we check the case of a new pristine Powergate. In this case, only the version will be set to the latest known version since doesn't make sense to run any migration.